src/op_handlers/near_call.rs

```diff
@@ -10,7 +10,7 @@ use crate::{opcode::Opcode, state::VMState};
 pub fn _near_call(vm: &mut VMState, opcode: &Opcode) {
     let abi_reg = vm.get_register(opcode.src0_index);
     let callee_address = opcode.imm0;
-    let _exception_handler = opcode.imm1; //TODO: Add exception handler to call frame
+    let exception_handler = opcode.imm1; //TODO: Add exception handler to call frame
```

Comment:
Exception handlers will live in the same contract since this is a near call, right? Wouldn't this simply be a PC modification to then jump to that routine? Can't we implement this here?

Reply:
I'm not sure what you mean, the exception handler is indeed what you say, a new PC where we want to jump, and that's whats being done

Reply:
I'll remove the TODO